toolchain-darwin-x86.txt

```diff
@@ -0,0 +1,45 @@
+# Copyright lowRISC contributors.
```

Comment:
How is this tested? If this isn't run in CI, and it breaks, we can't revert the change that broke it; you'll have to fix it yourself.

I am also generally concerned about trying to support non-Linux platforms, because there is non-trivial code where I have seen tacit Linux assumptions. What is our story for preventing breakage?

Reply:
Speaking only for myself, I'm ok merging PRs for experimental macOS support as long as we're careful not to break Linux.

Reply:
We should talk about how to create a CI for MacOS, and generally get a few folks working on the platform.  

Supporting more than one platform is always a concern, but in my experience I've found it makes a more robust product overall.  Let's come up with a plan for minimizing disruption.

Reply:
Until this is supported I'm concerned this file implies support that doesn't exist, especially because there aren't any caveats in README.md regarding unsupported experimental support.

We're reluctant to support this on the OT SW side because we're reluctant to commit to any sort of long term meson support.

Reply:
@drewmacrae are you ok with the PR otherwise? I can remove this file for now and place it in the riscv tools directory instead.

Reply:
toolchain file removed.